releasenotes/notes/add-ansible-vault-support-6c64f901599431d4.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Adds Ansible Vault support for decryption and encryption of
+    OpenStack secrets files during a leapfrog upgrade.  To enable:
+    export ANSIBLE_VAULT_PASSWORD_FILE=path_to_vault_pass_file
```

Comment:
Not that this breaks us, however the parser for releasenotes will see the colon as a new document. I'd recommend just quoting the command in back ticks.
  `export ANSIBLE_VAULT_PASSWORD_FILE=path_to_vault_pass_file`